mockup/patterns/structure/js/views/table.js

```diff
@@ -147,7 +147,7 @@ define([
                     // Clear the status message
                     self.app.setStatus();
                   });
-        self.app.setStatus(_t('Cannot drag and drop items to reorder while manually sorting a column'), 'warning', btn = btn);
+        self.app.setStatus(_t('Drag and drop not possible while non-permanent column sorting active.'), 'warning', btn = btn);
```

Comment:
@tkimnguyen is that proper english and is the message clear enough?
My problem with the message before was, that it could have suggested that the "manual column sorting" was permanently stored.

Reply:
@thet 
"Notice: Drag and drop reordering is disabled when viewing the contents sorted by a column"
and then next to the message the button to reset the sort is shown.